FirebaseFunctions/Tests/CombineUnit/HTTPSCallableTests.swift

```diff
@@ -16,56 +16,70 @@ import Foundation
 
 import Combine
 import FirebaseCore
+@testable import FirebaseFunctionsSwift
 import FirebaseFunctionsCombineSwift
-@testable import FirebaseFunctionsTestingSupport
+import FirebaseAuthInterop
+import FirebaseAppCheckInterop
+import FirebaseMessagingInterop
+import GTMSessionFetcherCore
 import XCTest
 
-// hardcoded in FIRHTTPSCallable.m
-private let kFunctionsTimeout: TimeInterval = 70.0
+// hardcoded in HTTPSCallable.swift
+private let timeoutInterval: TimeInterval = 70.0
 private let expectationTimeout: TimeInterval = 2
 
-class MockFunctions: Functions {
-  var mockCallFunction: () throws -> HTTPSCallableResult?
+@objc class MockFunctions: Functions {
```

Comment:
I'm curious if this is needed anymore after doing the right imports - I'm fine to leave it but don't want to have it here unnecessarily!

Reply:
Good catch! Removed.